src/neo/Network/P2P/RemoteNode.cs

```diff
@@ -29,6 +29,7 @@ internal class Relay { public IInventory Inventory; }
         public int ListenerTcpPort { get; private set; } = 0;
         public VersionPayload Version { get; private set; }
         public uint LastBlockIndex { get; private set; } = 0;
+        public uint LastHeightSent { get; private set; } = 0;
```

Comment:
Where is this field use?

Reply:
https://github.com/neo-project/neo/pull/1903/files#diff-a4af1b1da717ac3cf2434c8bb25eacf0R130-R131 in order to avoid send lower payloads already sent to this node.

Reply:
This will avoid the node to process and ping and respond with pong, right?

Reply:
Only will avoid duplicates